### ISSUE TITLE
Throws a container method undefined error

### ISSUE THREAD

**Issue by rizvfe00** [sensu-plugins/sensu-plugins-lxc#1]
Throws a container method undefined error using ruby 2.0.0p384 when running either of the shell scripts.

`$ check-lxc-status.rb
Check failed to run: undefined method `container' for LXC:Class, ["/var/lib/gems/2.0.0/gems/sensu-plugins-lxc-0.0.3/bin/check-lxc-status.rb:41:in `run'", "/var/lib/gems/2.0.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]`

**Comment by rizvfe00** [comment]
Changing this code inside the run method resolves this issue

`conn = LXC.container("#{config[:name]}")`
to

```
lxc = LXC.new
conn = LXC::Container.new(:lxc => lxc, :name => "#{config[:name]}")
```

**Comment by eheydrick** [comment]
@rizvfe00, can you open a PR with that change.

**Comment by rizvfe00** [comment]
Created PR
https://github.com/sensu-plugins/sensu-plugins-lxc/pull/2

**Comment by rizvfe00** [comment]
check-memstat.rb is pointing to a method that does not exist in the 0.6 lookout/lxc library (.memory_limit) and the status for whether a lxc container is running does not return back correctly from the library either.  

I opened an issue about 2 weeks ago on lookout/lxc but did not receive a response. I don't think that project is being actively maintained anymore.  Maybe this plugin would be better off using a newer library https://github.com/lxc/ruby-lxc ?

**Comment by mattyjones** [comment]
@rizvfe00 I would not be opposed to moving to that lib. If you want to code it up we would be happy to take a look at it.

**Comment by rizvfe00** [comment]
Sure, I'll create a new PR for it sometime this week.

**Comment by eheydrick** [comment]
Fixed in 1.0.0
